components/server/ee/src/prebuilds/github-service.ts

```diff
@@ -29,21 +29,19 @@ export class GitHubService extends RepositoryService {
         const octokit = await this.githubApi.create(user);
         const repositories = await octokit.paginate(
             octokit.repos.listForAuthenticatedUser,
-            { per_page: 100 },
+            { per_page: 100, admin: true },
```

Comment:
Looks useful! However, I've never seen this parameter documented anywhere:

- Not in Octokit: https://octokit.github.io/rest.js/v18#repos-list-for-authenticated-user
- Not in GitHub API: https://docs.github.com/en/rest/repos/repos#list-repositories-for-the-authenticated-user

Intrigued to see whether it works. 👀

Reply:
🤦🏻‍♂️  good point! my bad, the param's name is different. `type=owner` it is. 

The very request can be scoped ...

<img width="1104" alt="Screen Shot 2022-04-29 at 10 16 00" src="https://user-images.githubusercontent.com/914497/165911672-5069bd89-335f-4432-9fdd-d7bdd33fbdaa.png">

And the docs on the `type` param ... 

<img width="1369" alt="Screen Shot 2022-04-29 at 10 39 55" src="https://user-images.githubusercontent.com/914497/165911790-532b8f7b-99fb-41c7-a0ef-947b8776a412.png">

The `owner` type translates to `admin: true` on GitHub. 

Feel free to verify yourself.

Reply:
Thanks @AlexTugarev!

> The `owner` type translates to `admin: true` on GitHub.

Well, I initially did not use this, because I wasn't able to find the information that `owner` type (here) === `admin: true` for every repository.

Where can I find more documentation on the `type` parameter here?